repondeur/zam_repondeur/fetch/an/liasse_xml.py

```diff
@@ -99,8 +99,8 @@ def _make_amendement(node: etree.Element, uid_map: Dict[str, Amendement]) -> Ame
         DBSession,
         Amendement,
         lecture=lecture,
-        article=article,
         num=to_int(extract("identifiant", "numero")),
+        create_method_kwargs={"article": article},
```

Comment:
In #159 I move the assignment of `article` below, as it can change in an update.